arch/arm64/boot/dts/rockchip/rk3588-fxblox-rk1.dts

```diff
@@ -597,8 +641,8 @@
 
 &display_subsystem {
 	clocks = <&hdptxphy_hdmi_clk0>, <&hdptxphy_hdmi_clk1>;
-	clock-names = "hdmi0_phy_pll", "hdmi1_phy_pll";
-
+	clock-names = "hdmi0_phy_pll", "hdmi1_phy_pll";	
+	
```

Comment:
These two lines are adding unintended changes?

Reply:
no its ok

Reply:
What's the reason of adding extra spaces to these two line?

Reply:
I fixed it.